explainer/Explainer11.md

```diff
@@ -0,0 +1,287 @@
+
+```json
+{
+    "@context": ["https://www.w3.org/2022/wot/td/v1.1"], 
+    "@type" : "tm:ThingModel",
+    "title": "Lamp Thing Model",
+    "properties": {
+        "status": {
+            "description": "current status of the lamp (on|off)",
+            "type": "string",
+            "readOnly": true
+        }
+    },
+    "actions": {
+        "toggle": {
+            "description": "Turn the lamp on or off"
+        }
+    },
+    "events": {
+        "overheating": {
+            "description": "Lamp reaches a critical temperature (overheating)",
+            "data": {"type": "string"}
+        }
+    }
+}
+```
+Thing Model definitions are identified by the "@type": "tm:ThingModel". As the example shows, it does not provide details about a single Thing instance due to the lack of communication and security metadata. This specification presents a mechanism for deriving valid Thing Description instances from such Thing Model definitions. In addition, other design concepts are specified, including how to override, extend, and reuse existing Thing Model definitions. 
+
+
+
+## Non-Goals of Thing Description (TD)
+
+The following are explictly NOT goals of the TD specification:
+
+- Application- and domain-specific metadata vocabularies.
+   - TDs support the use of domain-specific vocabularies,
+     but no domain-specific vocabularies are defined as part of the specification.
+- Modification of existing protocols or security mechanisms.
+- Specifications of new protocols or security mechanisms.
+   - TDs are meant to describe existing practice, not prescribe new practices.
+
+TDs are also limited to protocols that have resources and access points that can
+be described with URLs and to payload types that have an IANA media Type.
+Structured payloads are supported but should be
+conceptually interconvertible with JSON.
+
+
+## Differentiation to Thing Description 1.0
+The main difference of this new specification from Thing Description 1.0 is that it includes the Thing Model concept (see above). This was only discussed in the Annex of the Thing Description 1.0 and was called Thing Description Template that time. 
+
+A new features is also possible to signalize that a Thing is following a specific profile such as the [W3C WoT Core Profile](https://w3c.github.io/wot-profile/). 
+
+In general, TD 1.1 specification describes a superset of the features defined in Thing Description 1.0. The new minor version update is also used to redefine some features more clearly and to add additional examples to make it easier to understand. 
+
+The complete change log with the new features and refinements can be found in the [change log section](https://w3c.github.io/wot-thing-description/#changes). 
+
+
+
+
+## Implementations
+
+Implementations were built by
+Bosch, Smart Things, ERCIM, Hitachi, Intel, Oracle, Panasonic, Fujitsu and Siemens (**list needs to be updated**).
```

Comment:
, among others.  See ... for the complete list.

Reply:
done